src/workerd/api/tests/BUILD.bazel

```diff
@@ -31,6 +31,7 @@ wd_test(
         "tail-worker-test-jsrpc.js",
         "websocket-hibernation.js",
     ],
+    generate_all_compat_flags_variant = False,
```

Comment:
Are the `generate_all_compat_flags_variant = False` in this file all to cut down on CI times?

Reply:
Nope, that's just a miss on my part from older versions, I'll just remove this and make the tests pass.
Good catch